x11-packages/libfcft/build.sh

```diff
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://codeberg.org/dnkl/fcft
 TERMUX_PKG_DESCRIPTION="A small font loading and glyph rasterization library"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="3.1.9"
+TERMUX_PKG_VERSION="3.3.1"
 TERMUX_PKG_SRCURL=https://codeberg.org/dnkl/fcft/archive/${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=4b7e3b2ab7e14f532d8a9cb0f2d3b0cdf9d2919b95e6ab8030f7ac87d059c2b6
+TERMUX_PKG_SHA256=f18bf79562e06d41741690cd1e07a02eb2600ae39eb5752eef8a698f603a482c
 TERMUX_PKG_AUTO_UPDATE=true
```

Comment:
Why wasn't `fcft` picked up for auto-updating by the workflow?

Reply:
Could different package name cause such issue? see fcft vs. libfcft https://repology.org/projects/?search=fcft

Reply:
I'll test it locally and write a auto-update function for it if necessary.